<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewProfileDialog</class>
 <widget class="QDialog" name="NewProfileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>155</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create a New Profile</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="profile_box">
     <property name="title">
      <string>Profile Information</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLineEdit" name="name_edit">
        <property name="toolTip">
         <string>Name of the profile</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>HOT Global</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_url">
        <property name="text">
         <string>URL</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="url_edit">
        <property name="toolTip">
         <string>URL of the profile.</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>http://example.com/</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_name">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewProfileDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>550</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewProfileDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>550</x>
     <y>365</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
